update flows samples to use v1 API (#2633)
knative/docs: docs/eventing/samples/parallel/mutual-exclusivity/README.md

exclusive branches.

This example is the same as the

> [multiple branches example](../multiple-branches/README.md) except that we are
> now going to rely on the Knative
> [switch](https://github.com/lionelvillard/knative-functions#switch) function to
> provide a soft mutual exclusivity guarantee.

Reviewer: ```suggestion
[multiple branches example](../multiple-branches/README.md) except that it relies on the Knative
[switch](https://github.com/lionelvillard/knative-functions#switch) function to
provide a soft mutual exclusivity guarantee.
```

### Create the PingSource targeting the Parallel object

> This will create a PingSource which will send a CloudEvent with
> `{"message": "Even or odd?"}` as the data payload every minute.

Reviewer: ```suggestion
This example creates a PingSource that sends a CloudEvent with
`{"message": "Even or odd?"}` as the data payload every minute.
```

### Inspecting the results

You can now see the final output by inspecting the logs of the

> `me-event-display` pods. Note that since we set the `PingSource` to emit every
> minute, it might take some time for the events to show up in the logs.

Reviewer: ```suggestion
`me-event-display` pods.

**NOTE:** The `PingSource` to emit every minute, so it may take some time for the events to show up in the logs.
```
